clients/admin-ui/src/features/taxonomy/components/TaxonomyCustomFieldsForm.tsx

```diff
@@ -0,0 +1,109 @@
+import { AntForm, AntFormInstance, AntInput, AntSelect } from "fidesui";
+import { isEmpty } from "lodash";
+
+import { useCustomFields } from "~/features/common/custom-fields";
+import FidesSpinner from "~/features/common/FidesSpinner";
+import { AllowedTypes } from "~/types/api";
+
+interface TaxonomyCustomFieldsFormProps {
+  customFields: ReturnType<typeof useCustomFields>;
+  formId: string;
+  form: AntFormInstance;
+}
+
+export const customFieldsFormBaseName = "customFieldValues";
+
+const TaxonomyCustomFieldsForm = ({
+  customFields,
+  formId,
+  form,
+}: TaxonomyCustomFieldsFormProps) => {
+  const {
+    idToAllowListWithOptions,
+    idToCustomFieldDefinition,
+    isEnabled,
+    isLoading,
+    sortedCustomFieldDefinitionIds,
+  } = customFields;
+
+  if (!isEnabled || sortedCustomFieldDefinitionIds.length === 0) {
+    return null;
+  }
+
+  return (
+    <AntForm
+      form={form}
+      name={formId}
+      initialValues={customFields.customFieldValues}
+      layout="vertical"
+      data-testid="custom-fields-form"
+    >
+      <h3 className="mb-3 font-semibold text-gray-700">Custom fields</h3>
```

Comment:
Yes! I've replaced this with Typography here and in the EditDrawer. Also while comparing the old vs new, I noticed the font was different because we never set ant to use the 'Inter' font. I've added a fontFamily token that to the ant.ts file, and I will double check this with jack.